### ISSUE TITLE
minimal.nix: deprecated option use for NixOS manual

### ISSUE THREAD

**Issue by ckauhaus** [NixOS/nixpkgs#54791]
## Issue description

When building configurations/images with minimal.nix, the warning appears:

```
warning: The option `services.nixosManual.enable' defined in `/home/local/ck/code/fc/nixpkgs-channels/nixos/modules/profiles/minimal.nix' has been renamed to `documentation.nixos.enable'.
````

### Steps to reproduce

`nix-instantiate nixos/release-combined.nix`

Tested on bc41317e24317b0f506287f2d5bab00140b9b50e

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.15.0-43-generic, Ubuntu, 18.04.1 LTS (Bionic Beaver)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.1.3`
 - nixpkgs: `/home/local/ck/code/fc/nixpkgs-channels`

**Comment by ckauhaus** [comment]
cc @oxij

**Comment by oxij** [comment]
This was fixed by 27982b408e465554b8831f492362bc87ed0ec02a. No longer reproducible on master.

**Comment by ckauhaus** [comment]
Thanks for having a look.
